include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -960,7 +960,13 @@ class CommonOperatorLoader : public ProtobufLoader {
       RETURN_IF_ERR(loadIdentity(op, dict));
       return true;
     }
-    if (typeName == "Identity") {
+    if (typeName == "EnforceFinite") {
+      // Load as Identity operation, because in Caffe2 this op does not change
+      // any data. It only raises error if some tensor element is Inf or NaN.
```

Comment:
> It only raises error if some tensor element is Inf or NaN.

like execution fails? or some data will be logged?

Reply:
C++ exception is thrown.
There is also an option to crash with fatal error, but it's disabled by default.
Here is exactly what happens:
https://github.com/pytorch/pytorch/blob/a4475d529d8a80c81fb0c1e0bc0f02f40ac53282/c10/util/Logging.cpp#L36